src/cljs/harja/views/urakka/yleiset/paivystajat.cljs

```diff
@@ -79,11 +79,14 @@
      {:otsikko "Alkupvm" :nimi :alku :tyyppi :pvm-aika :fmt pvm/pvm-aika :leveys 10
       :validoi [[:ei-tyhja "Aseta alkupvm"]
                 (fn [alku rivi]
-                  (let [loppu (:loppu rivi)]
+                  (let [loppu (:loppu rivi)
+                        id (:id rivi)]
                     (when (and alku loppu
                                (t/before? loppu alku))
-                      "Alkupvm ei voi olla lopun jälkeen.")))
-                ]}
+                      "Alkupvm ei voi olla lopun jälkeen.")
+                    (when (and (= id -1)
+                               (t/before? alku (t/today)))
```

Comment:
before? ottaa this (alku) ja (that) parametrit ja palauttaa true, mikäli this on ennen that, muuttujaa.
Eli nyt tulee "true" jos alku on ennen tätä päivää. Mutta kommentissa sanotaan ettei saa olla menneisyydessä.

Eli onko nuo muuttujat väärin päin before? funkkarilla, vai onko kommentti väärin?

(before? [this that] "Returns true if DateTime 'this' is strictly before date/time 'that'.")

Reply:
Teksti ei oo kommentti, vaan virheilmoitus käyttäjälle eli linjassa. Jos alku on ennen tätä päivää, alku on menneisyydessä  ja siitä tulee herja.